workshop/content/docs/setup/index.md

```diff
@@ -6,155 +6,169 @@ title: Setup of the workshop environment
  
 In this workshop we use the following materials:
 
-1) **Documentation** - (like this page): access latest on [dive.pygeoapi.io](https://dive.pygeoapi.io)
+1. **Documentation** - (like this page): access latest on [dive.pygeoapi.io](https://dive.pygeoapi.io)
+1. **Exercises** - [download this zip file](https://github.com/geopython/diving-into-pygeoapi/archive/refs/heads/main.zip), unzip, find exercises under `workshop/docker` [^1]
+1. **Docker** - all examples/exercises are run in a `docker container` 
 
-2) **Exercises** - [download this zip file](https://github.com/geopython/diving-into-pygeoapi/archive/refs/heads/main.zip), unzip, find exercises under `workshop/docker`
- 
-3) **Docker** - all examples/exercises run using Docker (and Docker Compose). 
-
-ad 2) alternatively you can fork/clone the GitHub repository of this workshop from https://github.com/geopython/diving-into-pygeoapi.
+[^1]: alternatively you can fork/clone the GitHub repository of this workshop from https://github.com/geopython/diving-into-pygeoapi.
 
-The main requirement is to install Docker and/with Docker Compose on your system.
-We strongly advise to do the Docker installation before the workshop starts.
+The main requirement for the training is, to install docker and/with `docker Compose` on your system.
+We strongly advise to install docker before the workshop starts.
 
 Although several custom installation-methods for `pygeoapi` are available and well-documented 
-at [pygeoapi.io](https://pygeoapi.io), these
-will *not* be considered in this workshop. 
-Exercises will also be based on Docker, hence a custom installation would at least be 'challenging'. 
-The good news that only a single installation (Docker) is needed! The Docker Images used
+at [pygeoapi.io](https://pygeoapi.io), these will *not* be considered in this workshop. 
+Exercises will also be based on docker, hence a custom installation would at least be 'challenging'. 
+The good news is that only a single installation (docker) is needed! The docker images used
 will contain the latest `pygeoapi` and all its dependencies and external services like PostGIS.
 
-## About Docker
+## About docker
 
 Docker has been available for almost 10 years, widely used on OSGeo software, so chances are big that you have
-heard of Docker and possibly of 'Containerization' in general. Or, better you are familiar and hopefully using Docker already.
+heard of docker and possibly of `containerization` in general. Or, better you are familiar and hopefully using docker already.
 If not, there is a plethora of introductory materials like [here](https://www.ibm.com/in-en/cloud/learn/docker).
-[Docker.com](https://www.docker.com/) remains the main entry.
+[docker.com](https://www.docker.com/) remains the main entry.
 
-Especially for the deployment and integration of Open Source Geospatial components, Docker
+Especially for the deployment and integration of Open Source Geospatial components, docker
```

Comment:
IMO Docker should start with capital if denoting the product or technology 'Docker'.

Reply:
makes sense, let's go for that, will update